htdocs/install/mysql/migration/17.0.0-18.0.0.sql

```diff
@@ -100,3 +100,19 @@ ALTER TABLE llx_recruitment_recruitmentcandidature ADD COLUMN fk_user integer;
 
 ALTER TABLE llx_bordereau_cheque ADD COLUMN type VARCHAR(6) DEFAULT 'CHQ';
 
+-- Element time
+ALTER TABLE llx_projet_task_time RENAME TO llx_element_time;
+ALTER TABLE llx_element_time CHANGE COLUMN fk_task fk_element integer NOT NULL;
+ALTER TABLE llx_element_time CHANGE COLUMN task_date element_date date;
+ALTER TABLE llx_element_time CHANGE COLUMN task_datehour element_datehour datetime;
+ALTER TABLE llx_element_time CHANGE COLUMN task_date_withhour element_date_withhour integer;
+ALTER TABLE llx_element_time CHANGE COLUMN task_duration element_duration double;
+ALTER TABLE llx_element_time ADD COLUMN elementtype varchar(32) NOT NULL DEFAULT 'task' AFTER fk_element;
+
+DROP INDEX idx_projet_task_time_task on llx_element_time;
```

Comment:
To drop an index, you must use 2 lines (one for mysql, one for postgresql). See examples on top of file.
So
-- VMYSQL4.1 DROP INDEX nomindex on llx_table;
-- VPGSQL8.2 DROP INDEX nomindex;

Reply:
I dit update